tests/integration/config/plugins/integration-tests/data/tests/tests_take_ships.txt

```diff
@@ -0,0 +1,226 @@
+		# Reload this to make sure the UUID is well stored into the savefile.
+		call "Open Menu"
+		call "Load First Savegame"
+		call "Depart"
+		call "Land"
+		assert
+			"ship model: Star Barge" == 0
+			"ship model: Sparrow" == 0
+			"%TEST%: Switch: offered" == 1
+			"%TEST%: Check sparrow: offered" == 1
+
+
+test-data "Taking Ship With Outfits And Unconstrained Test Save"
+	category "savegame"
+	contents
+		pilot Bobbi Bughunter
+		date 16 11 3013
+		system Sol
+		planet Earth
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 70
+				bunks 3
+				"cargo space" 50
+				drag 2.1
+				"engine capacity" 40
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 130
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 20
+			outfits
+				"X1700 Ion Thruster"
+				"X1200 Ion Steering"
+				Hyperdrive
+				"Anti-Missile Turret"
+				"D14-RN Shield Generator"
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8 "Anti-Missile Turret"
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system Sol
+			planet Earth
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 70
+				bunks 3
+				"cargo space" 50
+				drag 2.1
+				"engine capacity" 40
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 130
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 20
+			outfits
+				"Blaster Turret"
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+				"D14-RN Shield Generator"
+				"Laser Rifle"
+				"Chipmunk Plasma Thruster"
+				"Chipmunk Plasma Steering"
+				"Hyperdrive"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system "Tania Australis"
+			planet Ingot
+		account
+			credits 0
+			score 400
+			history
+
+test-data "Taking Ship With Outfits And Unconstrained Test Missions"
+	category mission
+	contents
+		mission "%TEST%: Switch"
+			landing
+			on offer
+				# the ship is not here
+				take ship "Star Barge (Armed)"
+					"requires outfits"
+		mission "%TEST%: Switch 2"
+			landing
+			on offer
+				# incorrect ship name
+				take ship "Star Barge (Armed)" "Well defended"
+					unconstrained
+		mission "%TEST%: Switch 3"
+			landing
+			on offer
+				# correct, takes the other ship
+				take ship "Star Barge (Armed)"
+					unconstrained
+
+test "Taking Ship With Outfits And Unconstrained"
+	status active
+	description "Test taking variants ships with outfits, and with the unconstrained keyword or not."
+		# Create/inject the savegame and load it.
```

Comment:
```suggestion
	sequence
		# Create/inject the savegame and load it.
```

Reply:
BRUH FOR REAL THERE IS NO WARNING IF THIS HAPPENS?